app/controllers/guidance_controller.rb

```diff
@@ -10,4 +10,8 @@ def about_register_trainee_teachers
   end
 
   def dates_and_deadlines; end
+
+  def manually_registering_trainees
+    render(:manually_registering_trainees, layout: "guidance_markdown")
```

Comment:
I've specified the page here because I wanted to use a specific layout - maybe I don't need to add the controller action too...